actionlist/app/containers/ActionListContainer.js

```diff
@@ -1,17 +1,69 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   View,
+  DrawerLayoutAndroid,
 } from 'react-native';
 
 import Header from '../components/header';
 import DisplayList from '../components/display_list';
+import SideMenu from '../components/SideMenu';
+import { toggleDrawer } from '../actions/action_items';
+
+class ActionListContainer extends Component {
```

Comment:
Hm, yeah, this is looking like bad design on the part of the React Native Drawer component. Would you consider wrapping the drawer in a stateful component rendered by your connected component (to separate the component using Redux from any kind of state management)? So you'd have this component that renders a `StatefulDrawer` component and just passes the open/closed property to that component, which does the state/ref management (and heavily comment that component to explain why it breaks the conventions of the application).